pathod/pathoc.py

```diff
@@ -73,7 +73,7 @@ def __str__(self):
             parts.append("\tPubkey: %s bit %s" % (pk.bits(), t))
             s = certutils.SSLCert(i)
             if s.altnames:
-                parts.append("\tSANs: %s" % " ".join(s.altnames))
+                parts.append("\tSANs: %s" % b" ".join(s.altnames))
```

Comment:
Yeah, you're right, the format string should be `bytes` too, and if that's the case then everything in the parts array should be bytes.

Something like [this](https://travis-ci.org/mitmproxy/mitmproxy/jobs/136891877#L617) happens when we mix bytes/str.